include/unifex/config.hpp.in

```diff
@@ -86,3 +86,17 @@
 # define UNIFEX_DECLARE_NON_DEDUCED_TYPE(NAME, ...) typename NAME = __VA_ARGS__
 # define UNIFEX_USE_NON_DEDUCED_TYPE(NAME, ...) __VA_ARGS__
 #endif
+
+#include <type_traits>
+// GCC 8 does not support remove_cvref
+#if defined(__GNUC__) && !defined(__clang__) && __GNUC__ < 9 && !defined(__cpp_lib_remove_cvref)
+namespace std
+{
+    template< class T >
+    struct remove_cvref {
+        typedef std::remove_cv_t<std::remove_reference_t<T>> type;
+    };
+    template <typename T>
+    using remove_cvref_t = typename remove_cvref<T>::type;
+}
+#endif
```

Comment:
Master now defines `unifex::remove_cvref[_t]` and uses it exclusively instead of `std::remove_cvref_t`. Please rebase this PR on master and remove this. Thanks, @cor3ntin!